BP/common/culture/cultures/00_continental_celtic.txt

```diff
@@ -0,0 +1,591 @@
+			Rogellach Ruaidrí Ruarcc Rumann Ráthach Rígán Ríán Ródán Rónchú Rónán Rúadacán Rúadán Scandal Scandlán Scellán Scolaige Sechnassach Selbach Senchán Sinach
+			Slébíne Snedgus Sogan Suibne Sáerbrethach Sárán Séigíne Sétna Sétnae Taithlech Talorc Talorcán Temnén Tigernach Tipraite Tomaltach Tomás_Thomas Torccán Trenchad
+			Trengus Tressach Tuathalán Tálán Tóla Túathal_Tuathal Uallgarg Uargal Uargalach Uargus Ultán Áed Áedgen Áeducán Áedán Áilgenán Áindle Ánrothán Échtgal
+			Échtgus Éicnech Éochad Éogan_Eugene Éremón Érennach Óengus Óenucán Ólchobar Úamnachán
+		}
+		female_names = {
+			Amia Ablach Affraic Aileann Anlaith Aróc Aíbinn Barrdub Ben-Laigen Brigit Bé-Fáil Bébinn Caillech-Fhinnéin Cainnech Caisséne Catan Caíntigern Caírech
+			Cellach Ciar Cnes Coblaith Conchenn Condál Crínóch Cumman Der-bForgaill_Devorguilla Derbáil Dianaim Doriend Dub-Essa Dub-Lemna Dubchoblaig Dubgilla Dúinsech
+			Dúnlaith Echrad Eithne Ellbríg Euginia_Eugenia Faílenn Fedelm Finneacht Finnguala Flann Forbflaith Fíne Garb Gelgéis Gerróc Gnathnad Gormlaith
+			Lathir Lerben Lerthan Lígach Martha Muirenn Muirgel Máel-Mide Máel-Muire Mór Mór-Muman Nárbflaith Sadb Samthann Sebdann Selblaith Sinech Suaibsech
+			Sáerlaith Sétach Síthmaith Tailefhlaith Tailltiu Temair Tuathflaith Tómnat Uallach Uasal Áedammair Étaín Íte Órlaith Órnat
+		}
+		patronym_prefix_male = "dynnpat_pre_mac" # So "mac "
+		patronym_prefix_female = "dynnpat_pre_inghean" # So "inghean "
+		always_use_patronym = yes
+
+		pat_grf_name_chance = 30
+		mat_grf_name_chance = 15
+		father_name_chance = 5
+
+		pat_grm_name_chance = 30
+		mat_grm_name_chance = 15
+		mother_name_chance = 10
+
+		founder_named_dynasties = yes
+
+		ethnicities = {
+			5 = caucasian_blond
+			5 = caucasian_ginger
+			45 = caucasian_brown_hair
+			35 = caucasian_dark_hair
+		}
+	}
+
+	lusitanian = { # Ancient Lusitanian
+		graphical_cultures = {
+			northern_clothing_gfx
+			mediterranean_building_gfx
+			iberian_group_coa_gfx
+		}
+
+		color = { 0.7 0.4 0.3 }
+
+		male_names = {
+			Apano Arantonio Arco Cabruno Camalo Ciliua Eburus Labaecus Lancius Maelo Pisiri
+			Pucio Taio Talabus Tanginus
+		}
+		female_names = {
+			Aunia Boudica Boutia Camina Cessea Cilia Cilura Dobiteina Dutia Iunia Lubana
+			Maila Mantau Sunua Tuouta Ulea
+		}
+		dynasty_of_location_prefix = "dynnp_of"
+
+		# Chance of male children being named after their paternal or maternal grandfather, or their father. Sum must not exceed 100.
+		pat_grf_name_chance = 30
+		mat_grf_name_chance = 15
+		father_name_chance = 25
+
+		# Chance of female children being named after their paternal or maternal grandmother, or their mother. Sum must not exceed 100.
+		pat_grm_name_chance = 40
+		mat_grm_name_chance = 20
+		mother_name_chance = 10
+
+		founder_named_dynasties = yes
+
+		ethnicities = {
+			5 = caucasian_blond
+			5 = caucasian_ginger
+			45 = caucasian_brown_hair
+			35 = caucasian_dark_hair
+		}
+	}
+
+	gaulish = {
```

Comment:
This one can actually be removed, it is there for compatibility with another mod, but completely unused

Reply:
Gaulish?

Reply:
Fixed

Reply:
Yes, you figured it out 😃